### DISCUSSION TITLE
Files for flashing Arduino and video tutorial are gone

### DISCUSSION THREAD

**Discussion by Widdiful** [sanni/cartreader#291] [Need help]
I ordered these parts a while ago and they've slowly been coming in, and now I want to flash the Arduino and double-check the video tutorial for the V3 but they seem to have been wiped. Could these be hosted somewhere? Thank you.

**Comment by sanni** [comment]
Thanks for reporting the broken link. Video should be working again.
Also the releases to flash the Arduino are archived here: https://github.com/sanni/cartreader/wiki/Cart-Reader-V3#older-releasesfor-testing-newer-releases-will-work-with-v3-hardware-too

**Reply by Widdiful** [reply]
Thank you, I managed to build it and it all seems to be working great! However, I also just noticed that the video in the user guide is also privated.
